### ISSUE TITLE
Edit server monitor text

### ISSUE THREAD

**Issue by KingMuffin11** [tedztar/mcstatusbot#187]
Hi there, 

Is it possible to edit the text that the server monitor shows? It currently reads like this

![image](https://github.com/tedztar/mcstatusbot/assets/20543838/e04d162e-d8af-4749-9ad2-416dd8946b12)

But I would like to be able to modify the text so that is shows like this

![image](https://github.com/tedztar/mcstatusbot/assets/20543838/aa5c9059-6018-47af-8c40-c0ddf20c1312)

**Comment by RahulR100** [comment]
Currently this is not possible but this is an interesting idea. We will consider it for the January update next year. Merry Christmas!

**Comment by KingMuffin11** [comment]
@RahulR100 I have changed this to a feature request :)

**Comment by RahulR100** [comment]
Yup! Definitely still in the pipeline Ill add this to the development roadmap. It will take second priority to multi language support but we will get it out there!

**Comment by RahulR100** [comment]
Circling back to this. Now that multi-language support is added, this feature will be released in the upcoming July update.

**Comment by RahulR100** [comment]
Hey! I know this has been waiting for a while but I finally managed to get to it, and it is now fully released. You can use the `/indicators` command to customize both online and offline indicators for any server.
